govcd/vm_test.go

```diff
@@ -652,6 +652,48 @@ func (vcd *TestVCD) Test_AnswerVmQuestion(check *C) {
 	check.Assert(isMediaInjected(vm.VM.VirtualHardwareSection.Item), Equals, false)
 }
 
+func (vcd *TestVCD) Test_VMChangeRootDiskSize(check *C) {
+	if vcd.skipVappTests {
+		check.Skip("Skipping test because vapp was not successfully created at setup")
+	}
+	vapp := vcd.findFirstVapp()
+	existingVm, vmName := vcd.findFirstVm(vapp)
+	if vmName == "" {
+		check.Skip("skipping test because no VM is found")
+	}
+
+	currentDisk := 0
+	if nil != vcd.vapp.VApp.Children.VM[0] && nil != vcd.vapp.VApp.Children.VM[0].VirtualHardwareSection && nil != vcd.vapp.VApp.Children.VM[0].VirtualHardwareSection.Item {
```

Comment:
Please use `variableName` compared to `value` and not the other way around.
If you run [staticcheck](https://staticcheck.io), you may get more similar advice

Reply:
This is again a consequence of copying the style from `Test_VMChangeCPUCountWithCore`. Would you prefer the staticcheck style, or consistency with the other tests?

Reply:
Just a few hours ago I updated go-vcloud-director with the removal of all these so called "Yoda comparisons". You may want to pull and merge from master again.

Reply:
Oh, excellent. I'll rebase my PR in addition to the style changes.